client/src/components/BottomBanner/index.tsx

```diff
@@ -26,6 +26,15 @@ const mapDispatchToProps = (dispatch: AppDispatch) => ({
   dispatch,
 });
 
+const getBannerContent = (surveyLink: string): JSX.Element => (
```

Comment:
wait, I think this can be a component 😄 ?

E.g.,

```tsx
function BannerContent({ surveryLink: string ) {
  return ...
}

...

<BannerContent surveyLink={surveyLink} />
```

Reply:
I've revised! thank you